model/src/wav_import_export.F90

```diff
@@ -753,14 +756,14 @@ subroutine export_fields (gcomp, rc)
       call CalcBotcur( va, wbcuru, wbcurv, wbcurp)
     end if
 
-    if ( state_fldchk(exportState, 'wavsuu') .and. &
```

Comment:
@JessicaMeixner-NOAA @alperaltuntas I think this section is not working as it is since it is using different variable names in if statement and get field pointer part. I am not sure this type coupling is tested under CESM or not but if it is running under CESM, I'll be surprised. So, I am not expecting any issue also in the CESM side.

Reply:
@uturuncoglu okay let me know if you need anything from my side.